flask_appbuilder/security/manager.py

```diff
@@ -255,6 +257,10 @@ def __init__(self, appbuilder):
             app.config.setdefault("AUTH_LDAP_LASTNAME_FIELD", "sn")
             app.config.setdefault("AUTH_LDAP_EMAIL_FIELD", "mail")
 
+        # Rate limiting
+        app.config.setdefault("AUTH_RATE_LIMITED", True)
+        app.config.setdefault("AUTH_RATE_LIMIT", "2 per 5 second")
```

Comment:
I don't think it's aggressive. A human being will have trouble meeting this if they have to retype the password and otherwise a short pause won't hurt. An automated system configured wrongly will remain wrong.

So if you realize "omg ive typed in the password/username twice" you probably start thinking a bit longer and cross the 5s barrier (try it!) and if you do make it a slight pause won't hurt IMHO.

An alternative is to redirect to a captcha when reaching the limit but I'll leave that to someone else.

Anyways I'm not married to this limit but I do not think it's too aggressive.

Reply:
I was able to reach it pretty easily and manually

Reply:
Easily as in normal human behavior or as in trying to trigger it? Triple entry isn't common. 

But if you think it's better we can switch.

Reply:
Trying to trigger it, but If you agree, I think it would be more conservative and still safe against brute force and DoS